examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -432,9 +435,82 @@ class ReadAll : public ReadCommand
 
     CHIP_ERROR SendCommand(chip::DeviceProxy * device, std::vector<chip::EndpointId> endpointIds) override
     {
-        return ReadCommand::ReadAll(device, endpointIds, mFabricFiltered);
+        return ReadCommand::ReadAll(device, endpointIds, mClusterIds, mAttributeIds, mEventIds, mFabricFiltered, mDataVersions,
+                                    mEventNumber);
+    }
+
+    void AddCommonArguments()
```

Comment:
> Common between what and what? That is, is there a reason this is not just inside the ReadAll constructor? Do we expect other constructors that would call this function but not the AddArgument for cluster-ids?

Initially I was hoping to add both this command to "any" but also to individual clusters but I end up not doing it since this is a bit complex to have something that makes sense based on the number of the individual arguments.
This is why there is a "common" method there. I can completely remove it if needed, but ideally it would be nice to have that for a given cluster too.

Reply:
It's fine to leave the common method.

Reply:
I end up removing it.